apps/wallet/src/ui/app/pages/site-connect/index.tsx

```diff
@@ -29,17 +32,18 @@ function SiteConnectPage() {
 	);
 	const dispatch = useAppDispatch();
 	const permissionRequest = useAppSelector(permissionSelector);
-	const activeAddress = useActiveAddress();
-	const [accountsToConnect, setAccountsToConnect] = useState<string[]>(() =>
-		activeAddress ? [activeAddress] : [],
+	const activeAccount = useActiveAccount();
+	const { data: accounts } = useAccounts();
+	const [accountsToConnect, setAccountsToConnect] = useState<SerializedUIAccount[]>(() =>
```

Comment:
We need to check if the account is unlocked before allowing the user to connect it. So we can't preselect it without checking that. We used to have a global lock but we removed it now

Reply:
I can follow up with this in the branch for locking / unlocking accounts

Reply:
Lets set it to empty array then and add a ticket and a TODO